src/Microsoft.SqlTools.ServiceLayer/EditData/Session.cs

```diff
@@ -0,0 +1,194 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Linq;
+using Microsoft.SqlTools.ServiceLayer.EditData.Contracts;
+using Microsoft.SqlTools.ServiceLayer.EditData.UpdateManagement;
+using Microsoft.SqlTools.ServiceLayer.QueryExecution;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.EditData
+{
+    public class Session
```

Comment:
>publ [](start = 4, length = 4)

Why internal? Also this is formatted like a property, not a field (TitleCase instead of camelCase) - why not at a minimum make a property?